patches/minecraft/net/minecraft/client/renderer/FogRenderer.java.patch

```diff
@@ -42,6 +78,16 @@
        if (fogtype == FogType.WATER) {
           float f = 192.0F;
           if (entity instanceof LocalPlayer) {
+@@ -199,7 +_,8 @@
+ 
+          RenderSystem.m_157445_(-8.0F);
+          RenderSystem.m_157443_(f * 0.5F);
+-      } else {
++      } else if (p_109025_.m_90593_() && !fluidState.m_76152_().getRegistryName().m_135827_().equals("minecraft") && p_109025_.m_90583_().f_82480_ < (double)((float)p_109025_.m_90588_().m_123342_() + fluidState.m_76155_(entity.f_19853_, p_109025_.m_90588_()))) net.minecraftforge.client.RenderProperties.get(fluidState).setupFog(p_109025_, p_109026_, p_109027_);
++      else {
```

Comment:
This would be more readable and not have any lines removed as part of this patch if it was:
```
} else if (thing) {
body
} else {
```